src/server/handlers/thirdpartyRequests/transactions/{ID}/authorizations.ts

```diff
@@ -42,6 +42,7 @@ import { Scope } from '../../../../../model/scope'
 import { consentDB, scopeDB } from '../../../../../lib/db'
 import { NotFoundError } from '../../../../../model/errors'
 import { verifySignature } from '../../../../../lib/challenge'
+import { thirdPartyRequest } from '../../../../../lib/requests'
 import { Request, ResponseToolkit, ResponseObject } from '@hapi/hapi'
 import {
   AuthPayload,
```

Comment:
move `domain` from `src/server` to `src` folder
in domain structure there is no need to use `{ID}` notation - it is required only for handlers